demos/asp-net-core/ITOps.ViewModelComposition/CompositionHandler.cs

```diff
@@ -0,0 +1,57 @@
+﻿using ITOps.ViewModelComposition.Engine;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ITOps.ViewModelComposition.Gateway
```

Comment:
As per https://github.com/Particular/Workshop.Microservices/pull/59#issuecomment-318675121, should this be moved to `ITOps.ViewModelComposition`?

Reply:
done